### DISCUSSION TITLE
Should PAGImageView.h remove NS_ASSUME_NONNULL_BEGIN declaration?

### DISCUSSION THREAD

**Discussion by ScriptBloom** [Tencent/libpag#2394] [Q&A] [answered]
I found that `PAGImageView.h` declare `NS_ASSUME_NONNULL_BEGIN` but `PAGView.h` not.

This may cause which I can call `[PAGView setPath:nil]` or `[PAGView setComposition:nil]`, but I cannot call `[PAGImageView setPath:nil]` or `[PAGImageView setComposition:nil]` to clean the resource pag of `PAGImageView`.

**Comment by domchen** [comment, marked as answer]
Thank you for your feedback. It has been removed: https://github.com/Tencent/libpag/pull/2456
